src/rom8002D6C.c

```diff
@@ -11,4 +14,191 @@ void sub_8002D70(struct Main * main)
     DmaCopy16(3, gUnknown_080150D0, gMain.roomData, sizeof(gUnknown_080150D0));
     DmaCopy16(3, gUnknown_08014FB8, gTalkData, sizeof(gUnknown_08014FB8));
     main->currentRoomId = 0;
-}
\ No newline at end of file
+}
+
+#define COPY_EXAM_DATA(data) DmaCopy16(3, data, gExaminationData, sizeof(data))
```

Comment:
i don't know about the name of this macro it loads that rooms examination data so COPY_EXAM_DATA sounds kinda misleading if you don't know what you're looking at i assume